docs/blog/testing-with-js-is-like-magic/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Testing with JS is like magic, but is it science?'
```

Comment:
Better as a subtitle

Reply:
The whole thing? Or after the comma? or?